unlock-app/src/helpers/Locks.js

```diff
@@ -1,6 +1,7 @@
 import configure from '../config'
 
-const config = configure(global)
+const isServer = typeof window === 'undefined'
+const config = !isServer ? configure(global) : {}
```

Comment:
I'd rather have the the `isServer` logic sit in `configure`. Could you move that there?

Reply:
Makes total sense 👍